### PR TITLE
Release 2023.12.2

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes for 2023.12.2
- **Is this PR related to a Monolith release?** Yes. 2023.12.2

### Resolves:

* [OKTA-677219](https://oktainc.atlassian.net/browse/OKTA-677219)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/a0e7a6eb3a37fa54311e06048c66e992a5c1e2f0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/34b88f65-20e9-467e-8be7-8242f026fcb0) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/a0e7a6eb3a37fa54311e06048c66e992a5c1e2f0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/35a705cd-b030-45c3-b5c5-95fb89caff94) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.07c92e52-12e5-4700-ad7e-d61981b7af81?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ddbb9cd38aa69e924a94745d50487c5fbeb8b7e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | 77 | 90 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2fb27cb9-5fa1-409b-9c05-d339dbd59ed9) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ddbb9cd38aa69e924a94745d50487c5fbeb8b7e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/14b4717c-6d0f-4cbe-bab2-b0cc8ff0a998) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/ddbb9cd38aa69e924a94745d50487c5fbeb8b7e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1045b915-bbc0-4fb1-8cb3-a29dfb1e00d5) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.17f6780c-af02-4400-a04b-791adadd21bd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ title: Okta Identity Engine API Products release notes 2023
 
 ## December
 
+### Weekly release 2023.12.2
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.12.2](#bugs-fixed-in-2023-12-2) | December 4, 2023 |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:15]: Should this date be January 4, 2024?

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ title: Okta API Products release notes 2023
 
 ## December
 
+### Weekly release 2023.12.2
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.12.2](#bugs-fixed-in-2023-12-2) | December 4, 2023 |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:13]: January 4, 2024?

**Review by grahamsmith-okta — APPROVED** [review]
Couple of date checks but otherwise good to go

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md]: Good catch! Thanks.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/488ffb9473573c11758d0ed98a4f9a88dfa98a8d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fokta-expression-language-in-identity-engine%2Findex.md) | 77 | 90 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7a931c83-8f53-4743-9f58-4194ed107c67) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/488ffb9473573c11758d0ed98a4f9a88dfa98a8d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d3104d98-7e39-4820-aff9-19438a39d702) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/488ffb9473573c11758d0ed98a4f9a88dfa98a8d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6816cd61-4383-4ac2-8a96-4b20372e70ec) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.406b91a7-9032-4349-b96f-365f556dd0da?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**